src/components/LabDetails/index.jsx

```diff
@@ -86,10 +88,26 @@ const LabDetails = ({
         </div>
       )}
       {!isLoading && labDetails && (
-        // Render the lab details here
-        <div>
-          {/* Render the details of labDetails */}
+      <div className="row">
+        <div className="col-12 col-md-7">
+          <LabDetailsCard
+            details={{
+              labProfileName: labDetails.LabProfileName,
+              userFirstName: labDetails.UserFirstName,
+              userLastName: labDetails.UserLastName,
+              startTime: formatTime(labDetails.StartTime),
+              endTime: formatTime(labDetails.EndTime),
+              state: labDetails.State,
+              completionStatus: labDetails.CompletionStatus,
+              totalRunTime: labDetails.TotalRunTime,
+              examPassed: labDetails.exam_passed ? 'Yes' : 'No',
+            }}
+          />
+        </div>
+        <div className="col-12 col-md-5">
+          {/* Render the pie chart card. */}
```

Comment:
```suggestion
          {/* Useful for further features. */}
```

Keep in mind that pie chart haven't been confirmed